LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
```

Comment:
- [ ] Kindly use an unordered list (ul) to list out your social icons, and wrap the icons in an `a` tag with href pointing to the social handle. This is best practice for HTML semantics and user experience.
![image](https://user-images.githubusercontent.com/94551484/219781926-17ec9443-10eb-4242-b21a-086f2ff73037.png)